### ISSUE TITLE
Image Title Help Text Describing Use As Alt Text

### ISSUE THREAD

**Issue by rgs258** [wagtail/wagtail#12069]
### Is your proposal related to a problem?

Yes, presently there is no indication to authors that the image's title will be used as it's alt text. I believe this fails to incentivize appropriately use the Title field.

### Describe the solution you'd like

I'd like to add help text.

### Describe alternatives you've considered

Adding an alttext field, but there be dragons.

### Additional context

Had a chat with @jacobtoppm and @vossisboss at Wagtail Space US

### Working on this

PR incomming.

**Comment by gasman** [comment]
Hi @rgs258! I believe [RFC 97](https://github.com/wagtail/rfcs/pull/97) (and the accompanying in-progress PR #11791) will address this.

**Comment by rgs258** [comment]
@gasman that was fast :)! Feel free to close this and the PR! As always, thank you for playing the game three steps ahead!

**Comment by gasman** [comment]
Closing in favour of #11791 👍
